ColorPickerPOC/ColorPickerPOC.js

```diff
@@ -90,6 +91,35 @@ const FavoriteColors = ({favoriteColors = [], favoriteColorsHandler, selectedCol
 		}, 100);
 	}, []);
 
+	const handleKeyDown = useCallback((ev) => {
```

Comment:
To avoid duplicated code I have this suggestion. I tested it and it seems to work. Please check it out as well.
1. Add an if to the onPressHandler to also deal with enter keydown like so:
```
const onPressHandler = useCallback((ev) => {
	if (editEnabled) return;
	if (ev.type === 'pointerdown' || (ev.type === 'keydown' && ev.keyCode === 13)) {
		const target = ev.target.id ? ev.target : ev.target.offsetParent;

		shakeEffectRef.current = setTimeout(() => {
			target.classList.add(componentsCss.shakeFavoriteColor);
		}, 300);

		timerRef.current = setTimeout(() => {
			setEditEnabled(true);
			setClickEnabled(false);
			target.classList.remove(componentsCss.shakeFavoriteColor);
		}, 1000);
	}
}, [editEnabled]);
```
2. For onReleseHandler you do not have to change anything.
3. Remove the two new handlers: handleKeyDown and handleKeyUp.
4. Use onKeyDown={onPressHandler} and onKeyUp={onReleaseHandler} for the spottable buttons.

I hope I did not mess things up. Thanks!